test/latest/Dockerfile

```diff
@@ -237,6 +237,9 @@ RUN install-tool gleam 1.12.0
 # renovate: datasource=github-releases packageName=kubernetes/kubernetes
 RUN install-tool kubectl v1.34.1
 
+# renovate: datasource=github-releases packageName=tofu/tofu
+RUN install-tool tofu 1.10.6
+
```

Comment:
move after skopeo

Reply:
Most of these are because I forgot to re-sort after changing from opentofu -> tofu. This set seems to be only alphabetical. Is that just an artifact of growth?